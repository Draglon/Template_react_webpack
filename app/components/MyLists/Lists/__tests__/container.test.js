import React from 'react';
import configureStore from 'redux-mock-store';

import ListsContainer from '../container';

describe('<ListsContainer />', () => {
  const props = {
    lists: [
      { id: 1, name: 'List 1', description: 'List 1 - description' },
      { id: 2, name: 'List 2', description: 'List 2 - description' },
    ],
  };

  const store = configureStore()({});
  const wrapper = shallow(<ListsContainer store={store} {...props} />);
  const container = wrapper.dive();

  it('should match snapshot', () => {
    expect(container).toMatchSnapshot();
  });
});
