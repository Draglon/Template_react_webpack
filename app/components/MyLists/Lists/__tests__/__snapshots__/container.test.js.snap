// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListsContainer /> should match snapshot 1`] = `
<ListsComponent
  deleteListRequest={[Function]}
  lists={
    Array [
      Object {
        "description": "List 1 - description",
        "id": 1,
        "name": "List 1",
      },
      Object {
        "description": "List 2 - description",
        "id": 2,
        "name": "List 2",
      },
    ]
  }
  onConfirm={[Function]}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
/>
`;
